tests/__init__.py

```diff
@@ -13,6 +13,9 @@
 os.environ['REDASH_GOOGLE_CLIENT_SECRET'] = "dummy"
 os.environ['REDASH_MULTI_ORG'] = "true"
 
+# Disable login rate limit skip
+os.environ['REDASH_SKIP_LOGIN_RATE_LIMIT'] = "false"
```

Comment:
This one is for the `test_throttle_login` not to fail during tests, but LMK if you think it's better to remove the test or sth else.

Reply:
`test_throttle_login` tests the feature but assumes configuration.
Imo it should be split into 3 tests:
1. Check that feature is configured on the server in a certain way (true, 50/hour).
2. Expect 429 error when feature is enabled.
3. Expect no error when feature is disabled.

Thus, setting the env var won't be located here, but inside the tests 2 and 3.

Reply:
> arikfr marked this conversation as resolved.

@arikfr ? :man_shrugging:

Reply:
Arik: "I'll click resolve and run away before anyone notices" 💨